llmfoundry/utils/model_download_utils.py

```diff
@@ -221,16 +232,18 @@ def download_from_oras(model: str,
                        config_file: str,
                        credentials_dir: str,
                        save_dir: str,
+                       tokenizer_only: bool,
```

Comment:
```suggestion
                       tokenizer_only: bool = False,
```
for consistency with `download_from_hf_hub`